core/src/main/java/nl/nn/adapterframework/util/CompactSaxHandler.java

```diff
@@ -28,7 +28,7 @@
 /**
  * SAX2 event handler to compact XML messages.
  *
- * @author  Peter Leeuwenburgh
+ * @author Peter Leeuwenburgh
  */
 public class CompactSaxHandler extends DefaultHandler {
```

Comment:
Poeh wat zit dit knullig in elkaar zeg, het zou mooier zijn als we dit gelijk streaming gaan maken.
Kijk anders is naar de FullXmlFilter en de vele implementaties er van.
In plaats van alles naar een charbuffer schrijven kunnen we dan aan een temp file oid schrijven. Misschien dat het handig is hier ook even met @tnleeuw naar te kijken, hoe we ContentHandler kunnen omtoveren tot een SerializableFileReference.

Reply:
Of echt streaming, ipv een temp file on disk.

Reply:
Volgens mij schrijf hij naar een writer weg, dus dan is een temp file beter. Als je het echt mooi wilt doen moet je iets maken dat kijkt naar de grote van de input als die beschikbaar is, dan kan je er voor kiezen het in memory te houden, of naar een tempfile te schrijven.